packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.template.ts

```diff
@@ -0,0 +1,65 @@
+import { html, when } from "@microsoft/fast-element";
+import { BreadcrumbItem } from "./breadcrumb-item";
+import { Breadcrumb } from "../breadcrumb";
+import { endTemplate, startTemplate } from "../patterns/start-end";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#(BreadcrumbItem:class)} component.
+ * @public
+ */
+export const BreadcrumbItemTemplate = html<BreadcrumbItem>`
+    <div role="listitem" class="listitem" part="listitem">
+        ${when(
+            x => x.href && x.href.length > 0,
+            html<BreadcrumbItem>`
+                <a
+                    class="control"
+                    part="control"
+                    href="${x => x.href}"
+                    aria-atomic="${x => x.ariaAtomic}"
+                    aria-busy="${x => x.ariaBusy}"
+                    aria-controls="${x => x.ariaControls}"
+                    aria-current="${x => (x.isCurrent ? "page" : void 0)}"
+                    aria-describedBy="${x => x.ariaDescribedby}"
+                    aria-details="${x => x.ariaDetails}"
+                    aria-disabled="${x => x.ariaDisabled}"
+                    aria-errormessage="${x => x.ariaErrormessage}"
+                    aria-expanded="${x => x.ariaExpanded}"
+                    aria-flowto="${x => x.ariaFlowto}"
+                    aria-haspopup="${x => x.ariaHaspopup}"
+                    aria-hidden="${x => x.ariaHidden}"
+                    aria-invalid="${x => x.ariaInvalid}"
+                    aria-keyshortcuts="${x => x.ariaKeyshortcuts}"
+                    aria-label="${x => x.ariaLabel}"
+                    aria-labelledby="${x => x.ariaLabelledby}"
+                    aria-live="${x => x.ariaLive}"
+                    aria-owns="${x => x.ariaOwns}"
+                    aria-relevant="${x => x.ariaRelevant}"
+                    aria-roledescription="${x => x.ariaRoledescription}"
+                >
+                    ${startTemplate}
+                    <span class="content" part="content">
+                        <slot></slot>
+                    </span>
+                    ${endTemplate}
+                </a>
```

Comment:
@eisenbergeffect is there a way we can reference the `fast-anchor` template here instead of duplicating it?

Reply:
If we can use the `fast-anchor`, I would be happy with that.

Reply:
If it's identical, you should be ale to compose it by just doing this

```
${AnchorTemplate}
```

Reply:
It looks like we will not be able to use `fast-anchor` there are a few attributes inside `fast-anchor` that is different in `breadcrumb-item` anchor. For example, breadcrumb has aria-current set to an observable created for `breadcrumb-item`
`aria-current="${x => (x.isCurrent ? "page" : void 0)}"`

Reply:
@khamudom - why not map to the aria property? The following is part of aria-globals which gets mixed in:
```
    /**
     * Indicates the element that represents the current item within a container or set of related elements.
     *
     * {@link https://www.w3.org/TR/wai-aria-1.1/#aria-current}
     * @public
     * @remarks
     * HTML Attribute: aria-current
     */
    @attr({ attribute: "aria-current", mode: "fromView" })
    public ariaCurrent:
        | "page"
        | "step"
        | "location"
        | "date"
        | "time"
        | "true"
        | "false"
        | string;

```

Reply:
It seems like the template should be fine unless we're trying to get fancy :)

Reply:
yeah, everything works great. just need to figure out how to change `aria-current`. Will give your suggestion a go.

Reply:
You shouldn't need to do anything but remove `isCurrent` from the class and from the template.